packages/astro/src/@types/astro.ts

```diff
@@ -697,12 +697,17 @@ export interface AstroUserConfig {
 	devOptions?: never;
 
 	legacy?: {
-		/**
-		 * Enable components and JSX expressions in markdown
-		 * Consider our MDX integration before applying this flag!
-		 * @see https://docs.astro.build/en/guides/integrations-guide/mdx/
-		 * Default: false
-		 */
+	 /**
+		 * @docs
+		 * @name astroFlavoredMarkdown
+		 * @type {boolean}
+		 * @default `false`
```

Comment:
We can also provide a `@since` tag, which should get transformed into the Since component in docs.

```suggestion
		 * @default `false`
		 * @since 1.0.0-rc
```